Makefile

```diff
@@ -1,5 +1,34 @@
-docs: docs/*.rst docs/conf.py docs/Makefile stravalib/*.py  ## generate html docs
-	# -fMeET -o docs/api stravalib stravalib/tests/ stravalib/tests/unit stravalib/tests/functional # omit tests dir from build
-	$(MAKE) -C docs clean
-	$(MAKE) -C docs html
-	$(MAKE) -C docs linkcheck
+TESTDIR=tmp-test-dir-stravalib
+PYTEST_ARGS=stravalib/tests/unit stravalib/tests/integration
+
+## I still need to add code cov to this build
+help:
+	@echo "Commands:"
+	@echo ""
+	@echo "  install   install package in editable mode"
+	@echo "  test      run the test suite (including doctests) and report coverage"
+	@echo "  build     build source and wheel distributions"
+	@echo "  clean     clean up build and generated files"
+	@echo ""
+
+
+build:
+	python -m build .
+
+install:
+	python -m pip install --no-deps -e .
+
+test:
```

Comment:
This runs the tests in a temp directory. It removes the temp directory after they are run. This is useful because we should be running tests on the installed package rather than the flat files as things can go wrong in the installed package that we won't see until they are on pypi.